spec/screens/creator_spec.rb

```diff
@@ -5,7 +5,7 @@
   let (:ui)       {instance_double(UI).as_null_object}
   let (:db)       {instance_double(DB).as_null_object}
   let (:creator)  {described_class.new(ui, db)}
-  let (:contact)  {{"name" => "name", "address" => "address", "phone" => "123456", "email" => "email@mail.com", "notes" => "notes"}}
+  let (:contact)  {{"id" => 1, "name" => "name", "address" => "address", "phone" => "123456", "email" => "email@mail.com", "notes" => "notes"}}
```

Comment:
Where tests actually failing if the contact wouldn't have an `id`?
I can't find ones that do rely on it.